src/components/Header/Header.style.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.header`
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 32px;
```

Comment:
I guess this is not necessary when you use `align-items: center`

Reply:
I started to use `align-items: center` but I wasn't sure if applying it would be the same measurements as the design pixel perfect.